src/PostgREST/Query/SqlFragment.hs

```diff
@@ -260,7 +275,7 @@ fromJsonBodyF body fields includeSelect includeLimitOne includeDefaults =
   (if includeDefaults
     then "LATERAL (SELECT jsonb_agg(jsonb_build_object(" <> defsJsonb <> ") || elem) AS val from jsonb_array_elements(pgrst_uniform_json.val) elem) pgrst_json_defs, "
     else mempty) <>
-  "LATERAL (SELECT * FROM " <>
+  "LATERAL (SELECT " <> parsedCols <> " FROM " <>
```

Comment:
@aljungberg This is the main change from https://github.com/PostgREST/postgrest/issues/2310. Had to be done bc of:

- https://github.com/PostgREST/postgrest/pull/2672
- https://postgrest.org/en/stable/references/api/tables_views.html#bulk-insert-with-default-values